connecpy.code-workspace

```diff
@@ -0,0 +1,60 @@
+{
+  "folders": [
+    {
+      "name": "/",
+      "path": "."
+    },
+    {
+      "name": "conformance",
```

Comment:
I realized I need a workspace file to be able to run tests in IDE in projects like noextras and now conformance. I went ahead and checked it in, which actually I always do in my own [projects](https://github.com/wasilibs/go-re2/blob/main/go-re2.code-workspace) but let me know if you'd prefer not to and I'll gitignore it instead.

Reply:
I just tried vscode multi-root workspace for the first time. I feel it's really convenient. Thanks for adding this 👍